release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -371,6 +371,13 @@ Package Manager CLI (`opm`), see the link:https://docs.okd.io/latest/operators/u
 [id="ocp-4-4-bug-fixes"]
 == Bug fixes
 
+*Web Console*
+
+* Earlier, when two consecutive rollouts failed, the *Topology* view showed failed pods instead of displaying the last active revision. With this bug fix, when a rollout fails, the last active revision is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760828[*BZ#1760828*])
+
+* Previously, when a user with limited cluster-wide permissions created an application in the *Add* page, by using the *Container Image* and *Image name from the internal registry* options, no imagestreams were detected in the project, though they existed. Now, the imagestream fetching logic has been moved from the cluster level to the namespace level, which enables a user with permission to a namespace to see imagestreams in that namespace.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1784264[*BZ#1784264*])
```

Comment:
The first sentence is still too long. Suggest breaking into 2-3 sentences, something like:
Previously, existing imagestreams were not detected when creating an application in the "Add" page. This occurred when users with limited cluster-wide permissions used the *Container Image* and *Image name from the internal registry* options.

Reply:
Thank you @bobfuru. I have tried to shorten all the big sentences in the PR. PTAL and let me know if anything else needs to be modified.

Reply:
> Thank you @bobfuru. I have tried to shorten all the big sentences in the PR. PTAL and let me know if anything else needs to be modified.

Thanks. The revisions look great. Two more small nits, if you don't mind changing?